README.md

```diff
@@ -711,6 +711,23 @@ colord("#e60000").isReadable("#ffff47", { level: "AAA", size: "large" }); // tru
 
 </details>
 
+<details>
+  <summary><b><code>.delta(color2 = "#FFF", options?)</code></b> (<b>lab</b> plugin)</summary>
+
+Calculates the perceived color difference between two colors.
+The difference calculated according to [Delta E2000](https://en.wikipedia.org/wiki/Color_difference#CIEDE2000).
+
+Returns the normalized value in range from [0, 1], where 0 is the same color and 1 are completely different.
+
+```js
+colord("#3296fa").delta("#197dc8") // 0.0989261
+colord("#faf0c8").delta("#fff") // 0.1447411
+colord("#afafaf").delta("#b4b4b4") // 0.0138414
+colord("#000").delta("#fff") // 1.0
```

Comment:
I would use our `round()` helper to make the result more consistent with other methods.


```suggestion
colord("#3296fa").delta("#197dc8") // 0.099
colord("#faf0c8").delta("#fff") // 0.145
colord("#afafaf").delta("#b4b4b4") // 0.014
colord("#000").delta("#fff") // 1
```